Automattic/pocket-casts-android
# Onboarding: recommendations trending section

| 📘 Project: #524  |
| --- |

## Description
Add the trending section to the recommendation screen

## Testing Instructions
1. Enable onboarding in `base.gradle` file. You may want to consider updating the startDestination [here](https://github.com/Automattic/pocket-casts-android/blob/2c6b39dccba6eb67aa643e17e59fdc3df59db437/modules/features/account/src/main/java/au/com/shiftyjelly/pocketcasts/account/onboarding/OnboardingFlowComposable.kt#L49) to be `OnboardingRecommendationsFlow.route`.
2. Open the onboarding recommendations screen
3. Observe that you can scroll through the trending podcasts for your region
4. Observe that tapping on any of the podcast subscribes/unsubscribes to that podcast

## Screenshots or Screencast 

https://user-images.githubusercontent.com/4656348/206279048-5405ab23-c988-4639-b519-8c4109f2d8de.mov

## Checklist
 
#### I have tested any UI changes...
